llvm/include/llvm/SandboxIR/Pass.h

```diff
@@ -12,11 +12,25 @@
 #include "llvm/Support/ErrorHandling.h"
 #include "llvm/Support/raw_ostream.h"
 
-namespace llvm::sandboxir {
+namespace llvm {
+
+class ScalarEvolution;
+
+namespace sandboxir {
 
 class Function;
 class Region;
 
+class Analyses {
+  ScalarEvolution *SE = nullptr;
+
+public:
+  /// Used by unittests.
+  Analyses() = default;
```

Comment:
I think this API is a bit dangerous to expose. Perhaps

```
class Analyses {
private:
Analyses() = default;
public:
Analyses(ScalarEvolution&) : SE(&SE) {}
static Analyses emptyForTesting() { return Analyses(); }
};
```

Reply:
Done